src/fromager/wheels.py

```diff
@@ -81,31 +85,65 @@ def _createenv(self) -> None:
         logger.info("installed dependencies into build environment in %s", self.path)
 
 
-def analyze_wheel_elfdeps(
-    ctx: context.WorkContext, req: Requirement, wheel: pathlib.Path
-) -> tuple[set[elfdeps.SOInfo], set[elfdeps.SOInfo]] | tuple[None, None]:
-    """Analyze a wheel's ELF dependencies
+def _iter_elfdeps(
+    req: Requirement,
+    wheel_root_dir: pathlib.Path,
+    settings: elfdeps.ELFAnalyzeSettings,
+) -> typing.Generator[elfdeps.ELFInfo, None, None]:
+    """Locate shared libraries and yield ELF information"""
+    for root, _, files in os.walk(wheel_root_dir):
+        rootpath = pathlib.Path(root)
+        for filename in files:
+            # any Python extension module or 'lib*.so*'
+            if filename.endswith(tuple(EXTENSION_SUFFIXES)) or (
+                filename.startswith("lib") and ".so" in filename
+            ):
```

Comment:
will we need this after https://github.com/python-wheel-build/elfdeps/pull/6 ?

Reply:
No, we don't need this any more. I have changed the code to use the new `analyze_dirtree` function.